packages/stencil/src/index.html

```diff
@@ -14,6 +14,15 @@
 <body>
   <datacom-menubar></datacom-menubar>
 
+<<<<<<< HEAD
```

Comment:
You've got merge conflicts in this file (evident by this HEAD line here). Please address.

Reply:
Fixed it.